RecoTracker/IterativeTracking/python/PixelPairStep_cff.py

```diff
@@ -36,6 +36,33 @@
     ]
 )
 
+_layerListForPhase2 = ['BPix1+BPix2', 'BPix1+BPix3', 'BPix2+BPix3',
+                       'BPix2+BPix4', 'BPix3+BPix4',
+                       'BPix1+FPix1_pos', 'BPix1+FPix1_neg',
+                       'BPix2+FPix1_pos', 'BPix2+FPix1_neg',
+                       'BPix1+FPix2_pos', 'BPix1+FPix2_neg',
+                       'FPix1_pos+FPix2_pos', 'FPix1_neg+FPix2_neg',
+                       'FPix2_pos+FPix3_pos', 'FPix2_neg+FPix3_neg',
+                       'FPix3_pos+FPix4_pos', 'FPix3_neg+FPix4_neg',
+                       'FPix4_pos+FPix5_pos', 'FPix4_neg+FPix5_neg',
+                       'FPix5_pos+FPix6_pos', 'FPix5_neg+FPix6_neg'
+]
```

Comment:
Same comment as https://github.com/makortel/cmssw/pull/2/files#r69396137, except if the phase1(PU70) and phase2 lists will be kept separate (it might well be better solution given that Phase1PU70 is only temporary), move the layer list inside the `toModify()`, i.e. remove the use of `_layerListForPhase2` variable (unless I'm missing some benefit from it).